cmd/editaccount.go

```diff
@@ -236,6 +243,21 @@ func (p *EditAccountParams) getTierLimits(tier int) (*jwt.JetStreamLimits, error
 	return src, nil
 }
 
+func (p *EditAccountParams) doDisableJetStream(r *store.Report) error {
+	if !p.disableJetStream {
+		return nil
+	}
+	p.claim.Limits.JetStreamLimits = jwt.JetStreamLimits{}
+	r.AddOK("deleted global limit")
```

Comment:
"delete" may be misinterpreted, I suggest "cleared global limits" and "cleared tiered limit"

Reply:
in other operations and tiers it is also labeled as "deleted" - so keeping it that way for now.